tailout/init.go

```diff
@@ -45,6 +53,11 @@ func (app *App) Init() error {
 		acl.TagOwners["tag:tailout"] = []string{}
 	}
 
+	if acl.AutoApprovers == nil {
+		fmt.Println("Auto approvers configuration does not exist.")
+		acl.AutoApprovers = &tsapi.ACLAutoApprovers{}
+	}
```

Comment:
I've added this check because I didn't have auto approvers in my config and it was throwing a panic.